tests/neg/i11170a.scala

```diff
@@ -0,0 +1,28 @@
+package apackage {
+
+  abstract class A {
+    protected val x: Int
+  }
+  abstract class A2 {
+    val x: Int
+  }
+}
+
+package bpackage {
+  import apackage._
+
+  trait B extends A {
+    println(x)
+  }
+  trait B2 extends A2 {
+    println(x)
+  }
+}
+
+package cpackage {
+  import apackage._
+  import bpackage._
+
+  case class C(override protected val x: Int) extends A with B  // error
```

Comment:
I think this should compile. "It's just a protected member." The ticket says it compiles if everything is in a single package.

Reply:
But Scala 2.13 also refuses this one. Is that a known bug?

Reply:
I see Scala 3 has a different definition of protected. It disalllows access to protected Java members from traits
```
illegal access to protected method test in class Base from trait Ext
```
at https://github.com/scala/bug/issues/3564

Maybe Scala just makes everything public if it wants to do something like that.

Reply:
The Scala 3 ticket was noticed at https://github.com/scala/bug/issues/12249 which is just Scala classes. The other link is a Java base class with protected member. I haven't reflected deeply yet.

Reply:
I am not sure about it either. But at least we don't crash anymore, and do the same thing as Scala 2.13 does.